collision_buffer_distance: 0.05
time_step_ms: 500
velocity_res: 0.05
yaw_rate_res: 0.075
max_yaw_rate: 0.5
spline_order: 3
goal_pos_tolerance: 0.1
goal_heading_tolerance: 0.2
goal_speed_tolerance: 0.1
timeout_ms: 1000
num_retries: 5

turtlebot_radius: 0.21
angular_col_res: 0.1
radial_col_res: 0.1
